app/controllers/sign_up_controller.rb

```diff
@@ -11,16 +12,30 @@ def create
   def login
     @user = User.find_by(email: params[:email])
     if @user&.authenticate(params[:password])
-      token = JsonWebToken.encode(name: @user.name, email: @user.email,
-                                  description: @user.description, phone: @user.phone,
-                                  gender: @user.gender, birthday: @user.birthday.to_s,
-                                  exp: 1.hour.from_now.to_i)
-      render json: { jwt: token }, status: :ok
+      if @user.email_confirmed
```

Comment:
I guess, you can use `@user.email_confirmed?` and it will also work.

Reply:
Fixed.